### PR TITLE
Increasing timeout for must-gather collection for ODF 4.11 and above

### LINKED ISSUE #6307: Increase must-gather collection timeout for ODF 4.11 and above
ocs-ci results for OCS4-11-Downstream-OCP4-11-AZURE-IPI-3AZ-RHCOS-3M-3W-tier1-or-tier_after_upgrade-post-upgrade (RUN ID: 1659901387)

https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster-prod/4962/consoleFull

1. tests/manage/z_cluster/cluster_expansion/test_node_expansion.py::TestAddNode::test_add_ocs_node

```
2022-08-08 15:27:33  09:57:32 - MainThread - ocs_ci.utility.utils - INFO  - Going to sleep for 15 seconds before next iteration
2022-08-08 15:27:48  09:57:47 - MainThread - ocs_ci.utility.utils - INFO  - Executing command: oc -n openshift-machine-api get machinesets j-001zi3c33-uba-cvtjg-app-eastus1 -n openshift-machine-api -o yaml
2022-08-08 15:27:48  09:57:47 - MainThread - ocs_ci.utility.utils - INFO  - Going to sleep for 15 seconds before next iteration
2022-08-08 15:28:04  09:58:02 - MainThread - ocs_ci.ocs.machine - ERROR  - New spun node failed to reach ready state OR Replica count didn't match ready replica count
2022-08-08 15:28:04  09:58:02 - MainThread - ocs_ci.utility.utils - INFO  - Executing command: oc -n openshift-machine-api get machinesets  -n openshift-machine-api -o yaml
2022-08-08 15:28:04  09:58:02 - MainThread - ocs_ci.utility.utils - INFO  - Executing command: oc -n openshift-machine-api describe MachineSet j-001zi3c33-uba-cvtjg-app-eastus1
2022-08-08 15:28:04  09:58:03 - MainThread - ocs_ci.ocs.utils - INFO  - Must gather image: quay.io/rhceph-dev/ocs-must-gather:latest-4.11 will be used.
2022-08-08 15:28:04  09:58:03 - MainThread - ocs_ci.ocs.utils - INFO  - OCS logs will be placed in location /home/jenkins/current-cluster-dir/logs/failed_testcase_ocs_logs_1659901387/test_add_ocs_node_ocs_logs/ocs_must_gather
2022-08-08 15:28:04  09:58:03 - MainThread - ocs_ci.utility.utils - INFO  - Executing command: oc adm must-gather --image=quay.io/rhceph-dev/ocs-must-gather:latest-4.11 --dest-dir=/home/jenkins/current-cluster-dir/logs/failed_testcase_ocs_logs_1659901387/test_add_ocs_node_ocs_logs/ocs_must_gather
2022-08-08 15:53:12  10:23:03 - MainThread - ocs_ci.ocs.utils - ERROR  - Timeout 1500s for must-gather reached, command exited with error: Command '['oc', 'adm', 'must-gather', '--image=quay.io/rhceph-dev/ocs-must-gather:latest-4.11', '--dest-dir=/home/jenkins/current-cluster-dir/logs/failed_testcase_ocs_logs_1659901387/test_add_ocs_node_ocs_logs/ocs_must_gather']' timed out after 1500 secondsMust-Gather Output: 
2022-08-08 15:53:12  FAILED
```

2. tests/manage/z_cluster/cluster_expansion/test_verify_ceph_csidriver_runs_on_non_ocs_nodes.py::TestCheckTolerationForCephCsiDriverDs::test_ceph_csidriver_runs_on_non_ocs_nodes
```
2022-08-08 16:08:19  10:38:19 - MainThread - ocs_ci.utility.utils - INFO  - Going to sleep for 15 seconds before next iteration
2022-08-08 16:08:37  10:38:34 - MainThread - ocs_ci.utility.utils - INFO  - Executing command: oc -n openshift-machine-api get machinesets j-001zi3c33-uba-cvtjg-app-eastus1 -n openshift-machine-api -o yaml
2022-08-08 16:08:37  10:38:34 - MainThread - ocs_ci.utility.utils - INFO  - Going to sleep for 15 seconds before next iteration
2022-08-08 16:08:49  10:38:49 - MainThread - ocs_ci.ocs.machine - ERROR  - New spun node failed to reach ready state OR Replica count didn't match ready replica count
2022-08-08 16:08:49  10:38:49 - MainThread - ocs_ci.utility.utils - INFO  - Executing command: oc -n openshift-machine-api get machinesets  -n openshift-machine-api -o yaml
2022-08-08 16:08:50  10:38:49 - MainThread - ocs_ci.utility.utils - INFO  - Executing command: oc -n openshift-machine-api describe MachineSet j-001zi3c33-uba-cvtjg-app-eastus1
2022-08-08 16:08:50  10:38:50 - MainThread - ocs_ci.ocs.utils - INFO  - Must gather image: quay.io/rhceph-dev/ocs-must-gather:latest-4.11 will be used.
2022-08-08 16:08:50  10:38:50 - MainThread - ocs_ci.ocs.utils - INFO  - OCS logs will be placed in location /home/jenkins/current-cluster-dir/logs/failed_testcase_ocs_logs_1659901387/test_ceph_csidriver_runs_on_non_ocs_nodes_ocs_logs/ocs_must_gather
2022-08-08 16:08:50  10:38:50 - MainThread - ocs_ci.utility.utils - INFO  - Executing command: oc adm must-gather --image=quay.io/rhceph-dev/ocs-must-gather:latest-4.11 --dest-dir=/home/jenkins/current-cluster-dir/logs/failed_testcase_ocs_logs_1659901387/test_ceph_csidriver_runs_on_non_ocs_nodes_ocs_logs/ocs_must_gather
2022-08-08 16:33:58  11:03:50 - MainThread - ocs_ci.ocs.utils - ERROR  - Timeout 1500s for must-gather reached, command exited with error: Command '['oc', 'adm', 'must-gather', '--image=quay.io/rhceph-dev/ocs-must-gather:latest-4.11', '--dest-dir=/home/jenkins/current-cluster-dir/logs/failed_testcase_ocs_logs_1659901387/test_ceph_csidriver_runs_on_non_ocs_nodes_ocs_logs/ocs_must_gather']' timed out after 1500 secondsMust-Gather Output: 
2022-08-08 16:33:58  FAILED
```

### PR DESCRIPTION
**Pull request by am-agrawa**
fixes #6307 

Signed-off-by: am-agrawa <amagrawa@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/utils.py [hunk 1]
```diff
@@ -887,7 +887,12 @@ def run_must_gather(log_dir_path, image, command=None):
     # https://github.com/red-hat-storage/ocs-ci/issues/3240
     mg_output = ""
     ocs_version = version.get_semantic_ocs_version_from_config()
-    timeout = 1500 if ocs_version >= version.VERSION_4_6 else 600
+    if ocs_version < version.VERSION_4_6:
```
Comment (keesturam) [inline ocs_ci/ocs/utils.py:890]: @am-agrawa thanks for sending this PR. 
could you please make this change

`timeout = 1800 if ocs_version >= version.VERSION_4_11 else 1500`

version below 4.6 is unsupported and we won't run any tests in those versions.

**Comment by am-agrawa** [comment]
Verified locally, for now it took
real    34m47.620s
user    1m39.291s
sys    0m28.660s

on OCP 4.12.0-0.nightly-2022-10-18-041406 and ODF 4.12.0-74

File: ocs_ci/ocs/utils.py [hunk 2]
```diff
@@ -887,7 +887,12 @@ def run_must_gather(log_dir_path, image, command=None):
     # https://github.com/red-hat-storage/ocs-ci/issues/3240
     mg_output = ""
     ocs_version = version.get_semantic_ocs_version_from_config()
-    timeout = 1500 if ocs_version >= version.VERSION_4_6 else 600
+    if version.VERSION_4_6 < ocs_version < version.VERSION_4_11:
+        timeout = 1500
+    elif ocs_version >= version.VERSION_4_11:
+        timeout = 2700
+    else:
+        log.error("OCS/ODF version isn't supported")
```
Comment (AviadP) [inline ocs_ci/ocs/utils.py:895]: if version is not supported it may be better to raise an exception instead of log error

**Comment by am-agrawa** [comment]
Closing this PR as Oded merged a fix- https://github.com/red-hat-storage/ocs-ci/pull/6587

**Pull request closed without merging**